src/components/combobox/combobox.tsx

```diff
@@ -880,6 +883,13 @@ export class Combobox implements LabelableComponent, FormComponent, InteractiveC
     } else if (activeItem.offsetTop < scrollTop) {
       this.listContainerEl.scrollTop = activeItem.offsetTop;
     }
+  };
+
+  shiftActiveItemIndex(delta: number): void {
+    const length = this.visibleItems.length;
```

Comment:
Nitpick: you could use destructuring if the assigned variable has the same name:

```
const { length } = this.visibleItems;
```